### DISCUSSION TITLE
Dynamically load highest minor version in shared scope

### DISCUSSION THREAD

**Discussion by ssmilin** [webpack/webpack#17391] [Q&A]
In a page, we are planning to load three apps in the shared scope. App 1 and App 2 are currently using React version 17.0.1 as their build version. Now, we intend to add App 3, which will use React version 17.0.2. However, there is a possibility that App 3 might inadvertently use the 17.0.1 version already present in the shared scope. Is there a way to dynamically load the highest minor version available in the shared scope during runtime?

**Comment by alexander-akait** [comment]
Are you looking for https://github.com/module-federation/module-federation-examples/blob/master/different-react-versions-isolated/README.md?
